setup.py

```diff
@@ -48,22 +48,30 @@
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
     install_requires = [
-        'Django==1.8.13',
-        'django-autocomplete-light==2.3.3',
-        'django-extensions>=1.5.5',
-        'django-nose>=1.4',
+        'Django>=1.8.2',
```

Comment:
This is kind of a symptom of combining the dnsaas project with django-powerdns-dnssec. The django-powerdns-dnssec app should absolutely not specify an exact Django version, it should allow all currently-supported versions of Django (currently 1.7+). The dnsaas project, on the other hand, isn't really a Django app, it's a full-blown Django *project*, and can therefore support whatever version/s of Django pass its tests.

Is there any reason why you don't want to allow users of django-powerdns-dnssec to bump their Django versions to 1.10?